src/types/index.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/ban-types */
 import type * as THREE from 'three'
-import { DefineComponent, Ref, VNode } from 'vue'
+import { DefineComponent, Ref, VNode, VNodeRef } from 'vue'
```

Comment:
```suggestion
import { DefineComponent, Ref, VNode } from 'vue'
```

Github Actions says it is not used 🤷‍♂️